### PR TITLE
Input for api calls

### PR DESCRIPTION
**Pull request by tomfelix**

### CODE REVIEW

**Comment by starspire** [comment]
@tomfelix your build failed. Fix linter issues.

File: src/App.tsx [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
-import { AppHeader, AppLogo, AppWrapper } from './components';
+import { AppHeader, AppHeaderLeft, AppHeaderCenter, AppHeaderRight, AppHeaderLogoLinkWrapper, AppLogo, AppHeaderNews, AppHeaderProof, AppWrapper } from './components';
 import { defaultTheme } from './theme/defaultTheme';
 
 import logo from './images/gardener-logo_horizontal.svg';
 import RequestList from './requests/RequestList';
 import ServerStatus from './status/ServerStatus';
+import CallForm from './customerCalls/CallForm';
+
+const url = 'https://gardeneroracle.io/';
```
Comment (lukaszblasz) [inline src/App.tsx]: better to have such urls in some kind of config in different file. For sure in project structure should be something like config.ts also such url can be places in .evn file

File: src/components/AppHeaderCenter.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const AppHeaderCenter = styled.div`
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 50%;
+    @media only screen and (max-width: 415px) {
```
Comment (lukaszblasz) [inline src/components/AppHeaderCenter.tsx]: 415px and const variable in some styled-components variables, also you can add it to theme object and read there from props.

File: src/components/AppHeaderLeft.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const AppHeaderLeft = styled.div`
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: space-around;
+    width: 25%;
+    height: 100%;
+    @media only screen and (max-width: 415px) {
```
Comment (lukaszblasz) [inline src/components/AppHeaderLeft.tsx]: Same as in above comment, also 415px should be a variable.

File: src/components/AppHeaderRight.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const AppHeaderRight = styled.div`
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 25%;
+    @media only screen and (max-width: 415px) {
```
Comment (lukaszblasz) [inline src/components/AppHeaderRight.tsx]: same here

File: src/customerCalls/CallForm.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        return (
+            <CallFormWrapper>
+                <CallFormInput
+                    value={this.state.query}
+                    onChange={this.handleChange}
+                    list="endpoints"
+                ></CallFormInput>
+                <CallFormDataList id="endpoints">
+                    <CallFormOption
+                        value={'json(https://api.coindesk.com/v1/bpi/currentprice.json).bpi.USD.rate_float'}
```
Comment (lukaszblasz) [inline src/customerCalls/CallForm.tsx]: All urls in this file should be taken from conts, ideally from different file.

File: src/customerCalls/components/CallFormWrapper.tsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+export const CallFormWrapper = styled.div`
+    font-family: ${props => props.theme.fonts.primary};
+    box-sizing: border-box;
+    max-height: 80px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    @media only screen and (max-width: 415px) {
```
Comment (lukaszblasz) [inline src/customerCalls/components/CallFormWrapper.tsx]: as in mutiple examples above, use variable for max-width

File: src/App.tsx (hunk 1 above)
Comment (tomfelix) [reply src/App.tsx]: created config.ts for urls

File: src/components/AppHeaderCenter.tsx (hunk 2 above)
Comment (tomfelix) [reply src/components/AppHeaderCenter.tsx]: 415px width use as variable now

File: src/components/AppHeaderLeft.tsx (hunk 3 above)
Comment (tomfelix) [reply src/components/AppHeaderLeft.tsx]: done

File: src/components/AppHeaderRight.tsx (hunk 4 above)
Comment (tomfelix) [reply src/components/AppHeaderRight.tsx]: done

File: src/customerCalls/CallForm.tsx (hunk 5 above)
Comment (tomfelix) [reply src/customerCalls/CallForm.tsx]: all urls moved to the config.ts file

File: src/customerCalls/components/CallFormWrapper.tsx (hunk 6 above)
Comment (tomfelix) [reply src/customerCalls/components/CallFormWrapper.tsx]: done

**Pull request merged**